src/components/navbar/Navbar.jsx

```diff
@@ -75,14 +79,14 @@ export const Navbar = (props) => {
             </div>
             <div
               onClick={() =>
-                loginHandler(loginWithGoogle, dispatch, authActions)
+                loginHandler(loginWithGoogle, dispatch, authActions, navigate)
```

Comment:
Suggestion - You don't need to pass navigate in loginHandler function, instead you can import it in whichever file you need

Reply:
since `loginHandler` is just a function, not a react custom hook or react component. can't import `useNavigate` in this case inside a function. Let me know if I'm wrong!